src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -367,181 +364,213 @@ private void testQuestionTypesSubmitAction(){
+        assertEquals("10 points left to distribute. The same amount of points should not be given multiple times.", submitPage.getConstSumMessage(qnNumber, 0));
         submitPage.fillResponseTextBox(qnNumber, 0, 0, "26");
-        assertEquals("9 points left to distribute.",submitPage.getConstSumMessage(qnNumber, 0));
+        assertEquals("9 points left to distribute.", submitPage.getConstSumMessage(qnNumber, 0));
         submitPage.fillResponseTextBox(qnNumber, 0, 0, "50");
-        assertEquals("Over allocated 15 points.",submitPage.getConstSumMessage(qnNumber, 0));
+        assertEquals("Over allocated 15 points.", submitPage.getConstSumMessage(qnNumber, 0));
         submitPage.fillResponseTextBox(qnNumber, 0, 1, "50");
-        assertEquals("Over allocated 25 points. The same amount of points should not be given multiple times.",submitPage.getConstSumMessage(qnNumber, 0));
+        assertEquals("Over allocated 25 points. The same amount of points should not be given multiple times.", submitPage.getConstSumMessage(qnNumber, 0));
         
-        //For other const sum question, just test one message.
+        // For other const sum question, just test one message.
         qnNumber = 18;
-        assertEquals("100 points left to distribute.",submitPage.getConstSumMessage(qnNumber, 3));
-        
+        assertEquals("100 points left to distribute.", submitPage.getConstSumMessage(qnNumber, 3));
     }
     
-    private void testContribSubmitAction(){
+    private void testContribSubmitAction() {
         ______TS("test submit actions for contribution questions.");
         
-        //No tests from instructor since contribution questions are only from students to own team members.
-        //Test by logging in as student instead.
-        
+        // No tests from instructor since contribution questions are only from students to own team members.
+        // Test by logging in as student instead.
         
-        //Test input disabled
+        // Test input disabled
         submitPage = loginToStudentFeedbackSubmitPage("Danny", "Closed Session");
+        
         int qnNumber = 1;
         int responseNumber = 0;
+        
         assertTrue(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         
+        // Test that the recipient selection is disabled and not visible
+        assertFalse(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
+        assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
     }
     
-    private void fillResponseTextBoxWithRecheck(int qnNumber, int responseNumber, String text, String expected) {
```

Comment:
I suspect this was a workaround for an intermittent test failure where selenium cannot fill the text box in the first attempt due to timing issues (e.g. selenium attempts to fill the text box before the browser renders the text box). May be it is no longer needed because newer Browswer/selenium versions do not have that problem. May be it is used only for slow computers or on a specific OS (Win/OS-X). Anyway, we can remove it first and see if it breaks anything.